Bugzilla/Attachment.pm

```diff
@@ -56,6 +56,7 @@ use base qw(Bugzilla::Object);
 ###############################
 
 use constant DB_TABLE   => 'attachments';
+use constant NAME_FIELD => 'name';
```

Comment:
This has surfaced something interesting -- I bet $attachment->name is never used and doesn't even make sense.

Reply:
Yup, this behaves no differently than it did before.

Reply:
If it is used, it will always be empty.

Reply:
Yes -- and by defined I mean as a database column -- so you can see in DB_COLUMS constant.
The way you've done this Attachment.pm is fine because you're replicating the existing behavior.